src/cmd/cmd_call_graph.ml

```diff
@@ -0,0 +1,86 @@
+(* SPDX-License-Identifier: AGPL-3.0-or-later *)
+(* Copyright © 2021-2024 OCamlPro *)
+(* Written by the Owi programmers *)
+
+open Syntax
+open Types
+open Runtime
```

Comment:
I think this `open Runtime` is not necessary as it is used only in one place. Could you remove it and write `Runtime.Local` were needed instead?